packages/arb-token-bridge-ui/tests/e2e/specs/txHistory.cy.ts

```diff
@@ -0,0 +1,147 @@
+import { startWebApp } from '../../support/common'
+
+const DEPOSIT_ROW_IDENTIFIER = /deposit-row-*/i
+const DEPOSIT_SEARCH_IDENTIFIER = /Search for a full or partial L1 tx ID/i
+const WITHDRAWAL_ROW_IDENTIFIER = /withdrawal-row-*/i
+const WITHDRAWAL_SEARCH_IDENTIFIER = /Search for a full or partial L2 tx ID/i
+
+describe('Transaction History', () => {
+  before(() => {
+    cy.changeMetamaskNetwork('goerli')
+    startWebApp()
+  })
+
+  after(() => {
+    cy.logout()
+  })
+
+  // Open tx history panel
+  it('should open transactions history panel', () => {
+    cy.openTransactionsPanel()
+    cy.findByText('Transaction History').should('be.visible')
+  })
+
+  // Test deposits are loading
+  it('should load deposits', () => {
+    cy.findByRole('tab', { name: 'show deposit transactions' })
+      .should('be.visible')
+      .click()
```

Comment:
Can we make it `click({ scrollBehavior: false })` instead? E.g. in CI we use a smaller screen size, might make some elements inaccessible.